test/widget/auth/email_auth_options_page_test.dart

```diff
@@ -329,50 +331,93 @@ void main() {
     });
 
     testWidgets(
-        'After input valid email/password/repeatPassword, createAccountButton dispatches SignUpWithEmail action',
+        'CreateAccountButton only dispatches when all textfields are valid',
         (WidgetTester tester) async {
-      // Setup the app state with expected values
-      final initialAppState = AppState.init();
-      final alteredState = initialAppState
-          .rebuild((b) => b..emailAuthOptionsPage.mode = EmailAuthMode.signUp);
-      final testMiddleware = VerifyDispatchMiddleware();
-
-      // Create the test harness.
-      final store = Store<AppState>(appReducer,
-          initialState: alteredState, middleware: [testMiddleware]);
+      // Setup a fake store and test harness with intial values.
```

Comment:
Effectively just different syntax for the same thing here, now using the FakeStore.

Reply:
shweet